gems/pending/appliance_console/database_replication_primary.rb

```diff
@@ -27,6 +27,11 @@ def ask_questions
       confirm
     end
 
+    def confirm
+      confirm_common
```

Comment:
Since these classes have a class/subclass can you just call the method in the superclass common and then `super` here?